src/index.scss

```diff
@@ -378,7 +385,7 @@ svg.svg-wrapper {
 }
 
 .outside-content-wrapper {
-  background-color: $darkBlue;
+  background-color: $purpleHathor;
```

Comment:
Did you try the Welcome Screen with a black background?

Reply:
I've just uploaded an image with this option. I think it's better as a background

Reply:
I also preferred it.